cmd/drone-server/config/config.go

```diff
@@ -166,6 +167,11 @@ type (
 		EnableAnonymousAccess bool `envconfig:"DRONE_PROMETHEUS_ANONYMOUS_ACCESS" default:"false"`
 	}
 
+	// Redis provides the redis configuration.
+	Redis struct {
+		ConnectionString string `envconfig:"DRONE_REDIS_CONNECTION"`
```

Comment:
I recommend we also include redis username and password as optional parameters.  If someone is using hosted redis (such as aws redis) they will need to provide the username and password to connect.